benchmarks/inference-server/text-generation-inference/variables.tf

```diff
@@ -70,6 +70,16 @@ variable "max_concurrent_requests" {
   }
 }
 
+variable "quantization" {
+  description = "Quantization used for the model."
+  type        = string
+  nullable    = true
+  # Can be one of the quantization options mentioned in https://huggingface.co/docs/text-generation-inference/en/basic_tutorials/launcher#quantize.
```

Comment:
small q, wondering if this is useful to add to the description itself?

we could also consider putting in validation as well with the options, ex: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/fdef210fbdfcb64e00e9fa17f6f346c2c3265ad6/benchmarks/inference-server/text-generation-inference/variables.tf#L55

Reply:
Yes, moving it into description makes sense. Updated. I didn't want to add validation since it requires us to keep it updated with any changes TGI makes.